.appveyor.yml

```diff
@@ -0,0 +1,46 @@
+# See https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/
+skip_branch_with_pr: true
+
+# AppVeyor will run on the output branch commits until the commit message
```

Comment:
Do you mean "unless" rather than "until"? Or are you saying CI runs on the initial empty commit like https://github.com/manubot/rootstock/commit/13115af95ca1b4daadd5242a4f3285557f8b87a8? It seems like our initial commits [do include](https://github.com/manubot/rootstock/blob/3defccfbea9d088f3b6885e1bf18158c6a05f467/SETUP.md#create-repository) `[ci skip]`:

```
git commit --allow-empty \
  --message "Initialize empty branch" \
  --message "[ci skip]"
```

Reply:
I meant "unless".  Fixed and expanded in 0817820